v1/lib/input/consoleinput.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
+
+require_once __DIR__."/../baseinput.php";
+require_once __DIR__."/../gamefield.php";
+
+/**
+ * This plugin takes input from the console to specify the width/height of the gamefield and which sets should be set alive.
```

Comment:
- This plugin asks the user over the console to ...
- which "sets" should be set alive? I would think that you wanted to say "which cells"